Source/AaxDecrypter/AaxDecrypter.csproj

```diff
@@ -13,7 +13,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="AAXClean.Codecs" Version="0.5.16" />
+	  <PackageReference Include="AAXClean.Codecs" Version="1.0.0" />
```

Comment:
w00h00 -- 1.0. Bet that feels good

Reply:
It does. I've done like 3 separate overhauls since version 0.1. I finally removed what I consider the last "hack" bits, so I think 1.0 is justified (: